README.md

```diff
@@ -19,27 +19,75 @@ kubectl create ns appmesh-system
 Apply the App Mesh CRDs:
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/aws/eks-charts/master/stable/appmesh-controller/crds/crds.yaml
+kubectl apply -f https://raw.githubusercontent.com/aws/eks-charts/master/stable/appmesh-manager/crds/crds.yaml
 ```
 
-Install the App Mesh CRD controller:
+Install the App Mesh controller:
 
 ```sh
-helm upgrade -i appmesh-controller eks/appmesh-controller \
+helm upgrade -i appmesh-manager eks/appmesh-manager \
 --namespace appmesh-system
 ```
 
-Install the App Mesh admission controller:
+The new AppMesh controller v1.0.0 is backwards-incompatible with older AppMesh controllers(<v1.0.0) and v1beta1
+AppMesh CRDs. If you have an old AppMesh controller/injector or v1beta1 AppMesh CRDs installed in your cluster, please
+follow the steps in the relevant upgrade sections below
+
+#### Upgrade without preserving old AppMesh resources
 
 ```sh
-helm upgrade -i appmesh-inject eks/appmesh-inject \
---namespace appmesh-system \
---set mesh.create=true \
---set mesh.name=global
+# Keep old AppMesh controller running, it is responsible to cleanup AppMesh resources in AWS
+# Delete all existing AppMesh CRs
```

Comment:
Can we please use "Custom resources" instead of CR as it may be confusing to some users

Reply:
ok